src/rebar_prv_escriptize.erl

```diff
@@ -143,7 +143,7 @@ escriptize(State0, App) ->
             {ok, #file_info{mode = Mode}} = file:read_file_info(Filename),
             ok = file:change_mode(Filename, Mode bor 8#00111);
         {win32, _} ->
-            write_windows_script(Filename)
+            write_windows_scripts(Filename, rebar_state:get(State, escript_wrappers_windows, []))
```

Comment:
should make it default to either the old script or both scripts, otherwise this won't ever generate anything for windows by default.

Reply:
Ah, yeah, good call :)